app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -17,6 +18,37 @@
   color: govuk-colour("white");
 }
 
-.gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:before {
+.gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:after {
```

Comment:
I'm still uncertain as to the value of shifting the chevron from `before` to `after`. In the screenshot in the PR description (assuming this is the 'after' state?) I'd be concerned about users not understanding that the link wrapped underneath (with no chevron) is part of the breadcrumb.

Reply:
That is the design that was decided on when discussions were had with TIm Paul

Reply:
Can you provide more detail of this? What was the justification?

Reply:
My confusion is that in the situation we're talking about, when there's only two breadcrumbs, there's only one chevron between them. To my mind it doesn't therefore matter whether it's a before or an after element - but my concern is that changing it from one to the other in order to override the styles from govuk-frontend adds extra lines of code that I don't understand the justification for. Does that make sense?